### ISSUE TITLE
show keyboard after on back press 

### ISSUE THREAD

**Issue by konuleminova** [adar2378/pin_code_fields#63]
It is not possible to show keyboard again , after pressing keyboard's on back button.
Please help to solve this issue. I already used autoDisposeControllers: false ,but it doesn't work.And problem is only for android devices.

**Comment by adar2378** [comment]
^ I thought it was solved here. Since you faced it, so it is still occurring I see

**Comment by adar2378** [comment]
Oh yours one a bit different, so when you close the keyboard by tapping the back button and try to refocus the textfield it doesn't open the keyboard?

**Comment by konuleminova** [comment]
@adar2378  Yes problem is this and it occurs only on android devices.After I tap keyboard on back button and then  try to focus textfield, but keyboard doesn't open.

**Comment by adar2378** [comment]
I've tested but it wasn't happening :| https://i.ibb.co/9GdqKNj/untitled.gif 
Can you tell me how to reproduce it?

**Comment by konuleminova** [comment]
After i have upgraded my flutter version to last ,this issue was happened. Which flutter version do you use? Did you check it in flutter latest version? 

See what happens in my phone after I tap back press.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/24622616/85871001-e02eaf00-b7de-11ea-9c1e-7e499070f7e8.gif)

**Comment by adar2378** [comment]
Flutter 1.17.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 1ad9baa8b9 (10 days ago) • 2020-06-17 14:41:16 -0700
Engine • revision ee76268252
Tools • Dart 2.8.4

This is the version I'm using. But the problem that I saw in your video, is not occurring for me. Which version are you using?

**Comment by byronkimani** [comment]
I was facing the same issue. Turns out I was using an older version of the package. Make sure you are using the latest package and pass the appContext to the PinCodeTextField. Hope this helps.
